src/features/movies/MoviePage/index.js

```diff
@@ -1,14 +1,95 @@
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useParams } from "react-router-dom";
+import { imageBaseUrl } from "../../../ApiValue";
 import { BigTile } from "../../../common/tiles/BigTile";
 import { Backdrop } from "./Backdrop";
-import { ContentContainer } from "./styled";
+import { fetchMoviePageDetails, selectMovie } from "./moviePageDetailsSlice";
+import {
+  fetchMoviePage,
+  selectMoviePageCast,
+  selectMoviePageCrew,
+} from "./moviePageSlice";
+import { Character, ContentContainer, Header } from "./styled";
+import {
+  Container,
+  Name,
+  ProfileImage,
+} from "../../../common/tiles/PersonTile/styled";
+import noPicture from "../../../common/tiles/PersonTile/noPicture.png";
+import { Wrapper } from "../MovieList/styled";
 
 export const MoviePage = () => {
+  const dispatch = useDispatch();
+  const { id } = useParams();
+
+  useEffect(() => {
+    dispatch(fetchMoviePage(id));
+    dispatch(fetchMoviePageDetails(id));
+  }, [dispatch, id]);
+
+  const movie = useSelector(selectMovie);
+  const cast = useSelector(selectMoviePageCast);
+  const crew = useSelector(selectMoviePageCrew);
+
   return (
-    <div>
+    <>
       <Backdrop />
       <ContentContainer>
-        <BigTile />
+        {movie.map(
+          ({
+            id,
+            poster_path,
+            title,
+            release_date,
+            production_countries,
+            vote_average,
+            vote_count,
+            overview,
+          }) => (
+            <BigTile
+              id={id}
+              poster_path={poster_path}
+              title={title}
+              release_date={release_date}
+              rating={vote_average}
+              votes={vote_count}
+              countries={production_countries}
+              article={overview}
+            />
+          )
+        )}
       </ContentContainer>
-    </div>
+      <Header>Cast</Header>
+      <Wrapper>
+        {cast.map(({ profile_path, name, character, id }) => (
+          <Container key={id} id={id}>
+            <ProfileImage
+              src={
+                profile_path ? `${imageBaseUrl}/w342${profile_path}` : noPicture
+              }
+              alt=""
+            ></ProfileImage>
+            <Name>{name}</Name>
+            <Character>{character}</Character>
+          </Container>
+        ))}
+      </Wrapper>
+      <Header>Crew</Header>
```

Comment:
tak jak przy personPage myślę, że warto dodać warunek logiczny czy w ogóle mamy crew i cast :)

Reply:
Poczekam jeszcze na opinie Radka, dalem narazie jeden warunek odnosnie wyswietlania crew w PersonPage jako odnosnik do tego czy chcemy je w naszym kodzie czy nie

Reply:
Osobiście zgodzę się z sugestią Madzi. Czyli lepiej będzie nie wyświetlać całej sekcji i tytułu gdy nie ma żadnych wartości.

Reply:
Ok to dodam, we wszystkich miejscach ten warunek